docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -213,6 +232,48 @@ following format:
   !<InstructionMetadata1> = !{<Operand1>, <Operand2>, ..}
   !<InstructionMetadata2> = !{<Operand1>, <Operand2>, ..}
 
++--------------------+---------------------------------------------------------+
```

Comment:
This table is not rendered in the rich diff

Reply:
Thanks for pointing this. Should be fixed by cfc7c8a8dc7ec4f45a530840c4a9472ea6a0af5f.

Reply:
This table is still not rendered when clicking "View file".

Reply:
It was broken again by https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/93ac5b2d110304b5975fcdfe4286b908910e7c11, but https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/16df0913330b960bcb118648c63c06e7579bb400 fixes it.